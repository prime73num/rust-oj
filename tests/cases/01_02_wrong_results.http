###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World.\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 423
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:25 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Wrong Answer","time":0}],"created_time":"2022-09-03T07:22:26.014Z","id":0,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World.\"); }","user_id":0},"updated_time":"2022-09-03T07:22:26.014Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { panic!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 423
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:26 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Runtime Error","time":0}],"created_time":"2022-09-03T07:22:26.291Z","id":1,"result":"Runtime Error","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { panic!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-03T07:22:26.291Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!{}\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 423
# content-type: application/json
# date: Sat, 03 Sep 2022 07:22:26 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Error","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-03T07:22:26.596Z","id":2,"result":"Compilation Error","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!{}\"); }","user_id":0},"updated_time":"2022-09-03T07:22:26.596Z"}
