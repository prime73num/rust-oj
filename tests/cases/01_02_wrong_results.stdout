{
  "id": 0,
  "created_time": "2022-09-03T04:02:04.841Z",
  "updated_time": "2022-09-03T04:02:04.841Z",
  "submission": {
    "source_code": "fn main() { println!(\"Hello World.\"); }",
    "language": "Rust",
    "user_id": 0,
    "contest_id": 0,
    "problem_id": 0
  },
  "state": "Finished",
  "result": "Wrong Answer",
  "score": 0.0,
  "cases": [
    {
      "id": 0,
      "result": "Compilation Success",
      "time": 0,
      "memory": 0,
      "info": ""
    },
    {
      "id": 1,
      "result": "Wrong Answer",
      "time": 0,
      "memory": 0,
      "info": ""
    }
  ]
}
{
  "id": 0,
  "created_time": "2022-09-03T04:02:05.150Z",
  "updated_time": "2022-09-03T04:02:05.150Z",
  "submission": {
    "source_code": "fn main() { panic!(\"Hello World!\"); }",
    "language": "Rust",
    "user_id": 0,
    "contest_id": 0,
    "problem_id": 0
  },
  "state": "Finished",
  "result": "Runtime Error",
  "score": 0.0,
  "cases": [
    {
      "id": 0,
      "result": "Compilation Success",
      "time": 0,
      "memory": 0,
      "info": ""
    },
    {
      "id": 1,
      "result": "Runtime Error",
      "time": 0,
      "memory": 0,
      "info": ""
    }
  ]
}
{
  "id": 0,
  "created_time": "2022-09-03T04:02:05.486Z",
  "updated_time": "2022-09-03T04:02:05.486Z",
  "submission": {
    "source_code": "fn main() { println!(\"Hello World!{}\"); }",
    "language": "Rust",
    "user_id": 0,
    "contest_id": 0,
    "problem_id": 0
  },
  "state": "Finished",
  "result": "Compilation Error",
  "score": 0.0,
  "cases": [
    {
      "id": 0,
      "result": "Compilation Error",
      "time": 0,
      "memory": 0,
      "info": ""
    },
    {
      "id": 1,
      "result": "Waiting",
      "time": 0,
      "memory": 0,
      "info": ""
    }
  ]
}
